src/popup/pages/AccountDetails.vue

```diff
@@ -167,16 +177,55 @@ export default defineComponent({
       }
     });
 
+    const showSearchBar = computed<boolean>(() => !!(
+      searchTerm.value || (
+        clientHeight.value > initialClientHeight.value
+        && appInnerScrollTop.value >= 120
+      )
+    ));
+
+    const resizeObserver = new ResizeObserver(debounce((entries) => {
+      if (!Array.isArray(entries) || !entries.length) {
+        return;
+      }
+      const _clientHeight = entries[0].target.clientHeight;
+      if (_clientHeight && (
+        (clientHeight.value + 400) < _clientHeight || _clientHeight <= initialClientHeight.value)
+      ) {
+        clientHeight.value = _clientHeight;
+      }
+    }, 100));
+    const routerBeforeEach = root.$router.beforeEach((to, from, next) => {
+      if (to.name && to.name.includes('account-details')) {
```

Comment:
In my opinion we need to avoid matching parts of the route names. At some point we need to store the route names as a constants and matching parts will block us from doing that. Also parts of route names are things hard to refactor/global search much harder.